panel.py

```diff
@@ -308,6 +308,38 @@ def __init__(self, name: str, fdt: Fdt2, node: int) -> None:
 		self.lp11_init = fdt.getprop_or_none(node, 'qcom,mdss-dsi-lp11-init') is not None
 		self.init_delay = fdt.getprop_uint32(node, 'qcom,mdss-dsi-init-delay-us')
 
+		# Display Stream Compression
+		self.compression_mode = fdt.getprop_or_none(mode_node, 'qcom,compression-mode')
+		self.has_dsc = self.compression_mode is not None and self.compression_mode.as_str() == "dsc"
+
+		if self.has_dsc:
+			#self.dsc_lm_split = fdt.getprop_or_none(mode_node, 'qcom,lm-split')
```

Comment:
Do we need this or not?

Reply:
Yesno. By default the kernel currently hardcodes a topology with two DSC encoders for a single interface because it's more power optimal to use two layer mixers and merge their output (lower clocks, because you have two hardware blocks ticking to perform this work):

```
- qcom,lm-split:			An array of two values indicating MDP should use two layer
					mixers to reduce power.
					Ex: Normally 1080x1920 display uses single DSI and thus one layer
					    mixer. But if we use two layer mixers then mux the output of
					    those two mixers into single stream and route it to single DSI
					    then we can lower the clock requirements of MDP. To use this
					    configuration we need two fill this array with <540 540>.
					Both values doesn't have to be same, but recommended, however sum of
					both values has to be equal to the panel-width.
					By default two mixer streams are merged using 2D mux, however if
					2 DSC encoders are used then merge is performed within compression
					engine.
```

However, for example the FP5 has a SoC with only one DSC encoder, so this feature has to be disabled (cc @lumag @z3ntu). Though:

1. We don't need this feature for that, we could use the catalog (and/or resource manager) to decide whether to disable it;
2. This feature hardcodes the size of both parts, but I don't have any DT utilizing it and no idea if it's supposed to work. Only one of my old old old kernels - and the deprecated `video` instead of `drm` driver while at it - parses it anyway.

Let's drop it.